src/Api/Controllers/AccountsController.cs

```diff
@@ -471,6 +471,20 @@ public async Task<ProfileResponseModel> PutProfile([FromBody] UpdateProfileReque
         return response;
     }
 
+    [HttpPut("avatar")]
+    [HttpPost("avatar")]
+    public async Task<ProfileResponseModel> PutAvatar([FromBody] UpdateAvatarRequestModel model)
```

Comment:
Is this taking in the entire user model and updating it to only update the avatar color? Would it make more sense to only send the userId and avatar color instead of the whole user profile?